frontend/src/metabase/visualizations/components/settings/ChartSettingActionIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import CS from "metabase/css/core/index.css";
+import {
+  ActionIcon,
+  type ActionIconProps,
+  Icon,
+  type IconName,
+} from "metabase/ui";
+
+interface ChartSettingActionIconProps extends ActionIconProps {
+  icon: IconName;
+  onClick: (target: HTMLElement) => void;
```

Comment:
nit: I'd prefer it if we can keep `event` as the argument for `onClick` to be consistent with other click handlers

Reply:
I just removed it so it'll rely on the `ActionIcon` props